index.html

```diff
@@ -132,7 +132,7 @@ <h3>Multi-Post Stories <br> Gain+Glory</h3>
                         <li>Html</li>
                     </ul>
                     <button class="project-button" type="button">
-                        <a href="#">See Project</a>
+                        <a href="#portfolio">See Project</a>
```

Comment:
- [ ] kindly make sure all the content from the work section is deleted from the HTML file since you were supposed to render it dynamically using JS

Reply:
- [ ] @Ouail-01 great work implementing the changes requested however you still haven't removed the work section content from your index.html file, kindly check [this example project](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-2a099ca3b81cdfe23a8dc8eb7677c597dbb0996f90476b3dd497e0bc36fb0f69)
![image](https://user-images.githubusercontent.com/61408860/209084743-96042196-73d6-4d17-a650-40deed3ffa96.png)